contents/newsletter/growth-engineer-secrets.md

```diff
@@ -0,0 +1,90 @@
+---
+title: The secrets of growth engineers
```

Comment:
'How to think like a growth engineer' tested well in the poll. 'How growth engineers think' could work well, too.

![Screenshot 2024-09-17 at 14 04 37](https://github.com/user-attachments/assets/78815021-a2ac-4908-900d-716609221caa)